distribution/openrct2.d.ts

```diff
@@ -2125,8 +2125,16 @@ declare global {
     interface SpinnerWidget extends WidgetBase {
         type: 'spinner';
         text?: string;
+
+        /**
+         * If true, the user can click and hold the decrement and increment
+         * button for repeated calls.
+         */
+        isHoldable?: boolean;
```

Comment:
Why would we want to turn this off? Native OS spinners/steppers always allow holding down, AFAICT. The fact that OpenRCT2 doesn't is a misfeature IMO.

Reply:
You mentioned this in https://github.com/OpenRCT2/OpenRCT2/issues/14095#issuecomment-779663385
I was thinking about making the default, but if you think the option should just be removed entirely, I can do that.

Reply:
I think removing it entirely and always allowing holding them down is indeed best. It improves usability and consistency (including with native OS widgets).

Reply:
ok, done